DLM_code/Src/dlm-manage_data_aquisition.c

```diff
@@ -240,14 +241,16 @@ void store_new_data()
                 // add the param data to RAM
                 if (add_param_to_ram(param_node, bucket_node))
                 {
-                	// TODO error handling
-
-                	// for now, turn on the onboard LED
-                	HAL_GPIO_WritePin(GPIOB, GPIO_PIN_14, GPIO_PIN_SET);
+                	// TODO malloc error handling
 
+                	// for now, turn on the onboard LED (ld2, blue)
+                	HAL_GPIO_WritePin(GPIOB, GPIO_PIN_7, GPIO_PIN_SET);
```

Comment:
Is this for manual testing? Makes sense, there might be some other things to do once we hit a soft fault like that. Might be nice to explore the watchdog timers, if we have a malloc fail we could just hang out to let the watchdog reset us